src/app/pages/IdxPage/index.jsx

```diff
@@ -1,11 +1,61 @@
-import React from 'react';
+import React, { useContext } from 'react';
+import { node, string } from 'prop-types';
+import styled from 'styled-components';
+import { GEL_GROUP_4_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import MostReadContainer from '#containers/MostRead';
+import MostReadSection from '#containers/MostRead/section';
+import MostReadSectionLabel from '#containers/MostRead/label';
+import { ServiceContext } from '#contexts/ServiceContext';
+
+const IdxMostReadSection = styled(MostReadSection)`
+  /* To centre page layout for Group 4+ */
+  margin: 0 auto;
+  width: 100%; /* Needed for IE11 */
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN};
+  }
+`;
+
+const MostReadWrapper = ({ children }) => (
+  <IdxMostReadSection>
+    <MostReadSectionLabel />
+    {children}
+  </IdxMostReadSection>
+);
+
+const renderMostRead = mostReadEndpointOverride => (
+  <MostReadContainer
+    mostReadEndpointOverride={mostReadEndpointOverride}
+    columnLayout="twoColumn"
+    wrapper={MostReadWrapper}
+  />
+);
+
+MostReadWrapper.propTypes = {
+  children: node.isRequired,
+};
+
+const IdxPage = ({ mostReadEndpointOverride }) => {
+  const {
+    dir,
+    mostRead: { onIdxPage },
+  } = useContext(ServiceContext);
 
-const IdxPage = () => {
   return (
-    <main role="main">
+    <main role="main" dir={dir}>
```

Comment:
Don't think we need to pass the direction to the `main`. It will be in the root `html` https://github.com/bbc/simorgh/blob/latest/src/app/containers/Metadata/index.jsx#L96 once we import the Metadata Container https://github.com/bbc/simorgh/pull/6683